packages/@ama-sdk/core/jest.config.js

```diff
@@ -1,21 +1,13 @@
 const { getJestModuleNameMapper } = require('@o3r/dev-tools');
-const {resolve} = require('node:path');
+const defaultConfig = require('../../../jest.config.ut');
 
 /** @type {import('ts-jest/dist/types').JestConfigWithTsJest} */
 module.exports = {
```

Comment:
the `transform` field should not be something like:
```typescript
transform: {
  ...defaultConfig.transform,
  '^.+\\.tsx?$': [/*...*/]
}
```
to be future proof?

Reply:
no longer needed with the factory